media-gfx/drawpile/additional-files/Drawpile.rdef

```diff
@@ -0,0 +1,24 @@
+
+resource app_signature "application/x-vnd.Drawpile";
+
+resource(1) #'VICN' array {
```

Comment:
It looks like `resource(1) #'VICN' array {` should be replaced by `resource vector_icon {` (or maybe `resource vector_icon array {`.

I don't know if the word `array` should be used here. What I can tell is that most rdef files in our repo do not use it:

`*.rdef` or `*.rdef.in` files with :arrow_down: | Count
---:| ---:|
`resource vector_icon {` | 125
`resource vector_icon array {` | 34
Total | 159

Reply:
@bach5000, you need to make a couple of changes in the recipe for the rdef to be used:
```diff
 SOURCE_DIR="Drawpile-$portVersion"
+ADDITIONAL_FILES="Drawpile.rdef"
 
 ARCHITECTURES="!x86_gcc2 ?x86 x86_64"
 SECONDARY_ARCHITECTURES="?x86"
```
and
```diff
 INSTALL()
 {
-	# TODO Add a rdef icon for Drawpile
 	mkdir -p $appsDir/Drawpile
 	cp ./build/bin/drawpile $appsDir/Drawpile/Drawpile
 	cp ./build/bin/drawpile-srv $appsDir/Drawpile/drawpile-srv
+
+	addResourcesToBinaries $portDir/additional-files/Drawpile.rdef \
+		$appsDir/Drawpile/Drawpile
+
 	addAppDeskbarSymlink $appsDir/Drawpile/Drawpile
}

Reply:
@fbrosson. I just drew an icon in Icon-o-matic and exported it into rdef and that was what it gave me. If the format isn't correct, that should be an issue in Icon-o-matic.

Reply:
Oh. Didn't see your previous comment. I'll make those fixes too :)

Reply:
Actually I don't know if we're not supposed to edit the header line of rdef files created by Icon-O-Matic.
Anyway, I think we can (should?) remove the `(1)` because this is only useful if you want to assign a given index to the icon.
```diff
-resource(1) #'VICN' array {
+resource #'VICN' array {
```

@humdingerb, @Begasus, what header would you recommend here?
* `resource(1) #'VICN' array {`
* `resource #'VICN' array {`
* `resource vector_icon array {`
* `resource vector_icon {`

Thanks!